lib/kino/remote_execution_cell.ex

```diff
@@ -16,15 +16,18 @@ defmodule Kino.RemoteExecutionCell do
     {shared_cookie, shared_cookie_secret} =
       AttributeStore.get_attribute({@global_key, :cookie}, {nil, nil})
 
-    node = attrs["node"] || AttributeStore.get_attribute({@global_key, :node})
+    node = attrs["node"] || AttributeStore.get_attribute({@global_key, :node}) || ""
+    cookie_secret = attrs["cookie_secret"] || shared_cookie_secret || ""
+    cookie_secret_value = System.get_env("LB_#{cookie_secret}")
 
     fields = %{
       "assign_to" => attrs["assign_to"] || "",
-      "node" => node || "",
+      "node" => node,
       "cookie" => attrs["cookie"] || shared_cookie || "",
-      "cookie_secret" => attrs["cookie_secret"] || shared_cookie_secret || "",
+      "cookie_secret" => cookie_secret,
```

Comment:
If you do the above, you may need to do this:

```suggestion
      "cookie_secret" => cookie_secret || "",
```

Not sure.